packages/react/src/SearchControl/SearchControl.tsx

```diff
@@ -80,7 +81,8 @@ export interface SearchControlProps {
   onClick?: (e: SearchClickEvent) => void;
   onAuxClick?: (e: SearchClickEvent) => void;
   onNew?: () => void;
-  onExport?: () => void;
+  onExportCSV?: () => void;
+  onExportFHIRBundle?: () => void;
```

Comment:
Can we rename this to `onExportTransactionBundle`?

Here's my thinking:
1. We rarely (never?) use "fhir" in variable or function names.  It's often implicit.
2. `Bundle` is well established throughout the code base, so this should be self explanatory.
3. In future work (or maybe you could add it to this PR if it is easy), I think we should also include `Export Bundle` / `onExportBundle` which is the raw FHIR Bundle we receive from the API server.

To elaborate on that:

- "Export Bundle"
  - Exports the search Bundle exactly as is
  - `Bundle.type` = `searchset`
  - Keep original ID's
  - Keep original `meta`
  - Keep original `fullUrl`
- "Export Transaction Bundle"
  - What you implemented in this PR
  - `Bundle.type` = `transaction`
  - Remove original ID`s
  - Remove original meta
  - Add `urn:uuid:` to `fullUrl`